packages/panna-sdk/src/core/utils/activity.ts

```diff
@@ -287,6 +360,18 @@ export const getActivitiesByAddress = async function (
     activityCache.set(ckActivities, preProcessedActivities);
   }
 
+  // Fetch token prices for fiat value calculations
+  let tokenPrices: Array<{ symbol: string; prices: Record<string, number> }> =
```

Comment:
Define a type for `Array<{ symbol: string; prices: Record<string, number> }>` and reuse?